### ISSUE TITLE
Error saving configuration:  Could not locate configuration for 'statamic-chatgpt'.

### ISSUE THREAD

**Issue by euoia** [BitDigitalCodes/statamic-chatgpt#2]
I installed according to the instructions with `composer require bitdigitalcodes/statamic-chatgpt`

When attempting to save configuration details I got a server error. From inspecting the response payload:

```
{
    "message": "Could not locate configuration for 'statamic-chatgpt'.",
    "exception": "Stillat\\Proteus\\Exceptions\\ConfigNotFoundException",
    "file": "/path/to/mysite.com/vendor/stillat/proteus/src/LaravelConfigWriter.php",
    "line": 399,
    "trace": [
    ...
    ]
}
```

**Comment by mikemike** [comment]
Thanks for the report @euoia 

The issue is because the config hasn't been copied, you can fix this with:

```
php artisan vendor:publish
```

And then selecting: `Bitdigital\StatamicChatgpt\ServiceProvider` from the list.

You shouldn't have to do this, it should sort itself out as part of the installation process, but that's not currently happening so I'll get that fixed today.

**Comment by mikemike** [comment]
This is now fixed in #3 with release v1.0.2 (https://github.com/BitDigitalCodes/statamic-chatgpt/releases/tag/v1.0.2)
